editor/plugins/texture_region_editor_plugin.cpp

```diff
@@ -38,64 +38,67 @@
 #include "editor/editor_undo_redo_manager.h"
 #include "scene/gui/check_box.h"
 #include "scene/gui/option_button.h"
+#include "scene/gui/panel_container.h"
 #include "scene/gui/separator.h"
 #include "scene/gui/spin_box.h"
 #include "scene/gui/view_panner.h"
 #include "scene/resources/atlas_texture.h"
 
-void draw_margin_line(Control *edit_draw, Vector2 from, Vector2 to) {
-	Vector2 line = (to - from).normalized() * 10;
+void draw_margin_line(Control *p_texture_overlay, Vector2 p_from, Vector2 p_to) {
+	Vector2 line = (p_to - p_from).normalized() * 10;
 
 	// Draw a translucent background line to make the foreground line visible on any background.
-	edit_draw->draw_line(
-			from,
-			to,
+	p_texture_overlay->draw_line(
+			p_from,
+			p_to,
 			EditorNode::get_singleton()->get_theme_base()->get_theme_color(SNAME("mono_color"), SNAME("Editor")).inverted() * Color(1, 1, 1, 0.5),
 			Math::round(2 * EDSCALE));
 
-	while (from.distance_squared_to(to) > 200) {
-		edit_draw->draw_line(
-				from,
-				from + line,
+	while (p_from.distance_squared_to(p_to) > 200) {
+		p_texture_overlay->draw_line(
+				p_from,
+				p_from + line,
 				EditorNode::get_singleton()->get_theme_base()->get_theme_color(SNAME("mono_color"), SNAME("Editor")),
 				Math::round(2 * EDSCALE));
 
-		from += line * 2;
+		p_from += line * 2;
 	}
 }
```

Comment:
There's a lot of magic numbers used. I know that it was in the previous code, but could we use some `constexpr` to express those numbers?

Reply:
I can do another code quality pass as a follow up. There are more things that can be improved here, but I'd rather not dilute this PR too much — it's pretty messy as it is (I should've acted with more restraint to begin with).

Reply:
Maybe just for that function. It's pretty egregious. Like, what does the 200 mean on line 7? And the `* 2` afterwards.

Reply:
The `*2` is because it's a dashed line. We first draw a solid line then draw dashes on top of it with a contrasting color. `line` is the size of the dash, so we draw one unit of `line` and skip another unit of `line`.

As for 200, I'm not completely sure myself.

Reply:
I've done a little cleanup and added comments to explain the logic.